src/Components/MOCK_LEADERBOARD_DATA.json

```diff
@@ -0,0 +1,100 @@
+[{"id":1,"user_name":"ibirbeck0","total_score":54236},
```

Comment:
Next sprint we should limit the table to show the top 10, and add a search option.